gradle.properties

```diff
@@ -1,3 +1,3 @@
-version=1.2.4
+version=1.3.0
```

Comment:
this project uses semantic releases, so there´s no need to increase manually the version spec. Please undo this change.